math/maxima/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem 1.0
 
 name            maxima
-version         5.39.0
-revision        14
+version         5.41.0
+revision        0
```

Comment:
Generally, we only include the `revision` line if the revision is 1 or above.

Reply:
I see. I will remove the revision line.

Reply:
After you have pushed the fixes for this and the other issues, it should be possible to merge this.

Reply:
Thank you.
But I need some more time to check 'sub-port'.